app/views/claims/support/schools/users/check.html.erb

```diff
@@ -1,7 +1,11 @@
 <%= content_for :page_title, sanitize(t(".page_title", school_name: @school.name)) %>
 
 <%= content_for(:before_content) do %>
-  <%= govuk_back_link(href: new_claims_support_school_user_path(@school)) %>
+  <%= govuk_back_link(
+    href: new_claims_support_school_user_path(
+      params.permit(claims_user: {}).merge(school_id: @school.id),
```

Comment:
Can we permit the params in the controller rather than in the view?

Reply:
I _think_ we can pass through something like `@user.attributes.slice(*attributes_we_want)`.

Reply:
I've done exactly that, but moved it to the decorator so it's not overly complicating the view.
resolved in [e56a8f2](https://github.com/DFE-Digital/itt-mentor-services/pull/136/commits/e56a8f27dbadac36b1c7b5800ec59c670bba23cb)